android/assets/jsons/translations/template.properties

```diff
@@ -591,6 +592,10 @@ You have to write an ID for your friend! =
 You cannot add your own player ID in your friend list! = 
 To add a friend, ask him to send you his player ID.\nClick the 'Add friend' button.\nInsert his player ID and a name for him.\nThen click the 'Add friend' button again.\n\nAfter that you will see him in your friends list.\n\nA new button will appear when creating a new\nmultiplayer game, which allows you to select your friend. = 
 Please input Player ID! = 
+Random civilizations = 
+Set values = 
+Please enter values for maxmimum and minimum number of civilizations! = 
+Please enter only integer values! = 
```

Comment:
This should be part of the textbox, it should only allow characters 0-9, there's a character filter there somewhere